src/test/java/seedu/address/commons/core/ConfigTest.java

```diff
@@ -20,8 +24,33 @@ public void toStringMethod() {
     public void equalsMethod() {
         Config defaultConfig = new Config();
         assertNotNull(defaultConfig);
-        assertTrue(defaultConfig.equals(defaultConfig));
+        assertEquals(defaultConfig, defaultConfig);
+        assertFalse(defaultConfig.equals(2));
+
+        Config otherConfig = new Config();
+        assertEquals(defaultConfig, otherConfig);
+
+        otherConfig.setLogLevel(Level.OFF);
+        assertNotEquals(otherConfig, defaultConfig);
+
+        otherConfig = new Config();
+        otherConfig.setUserPrefsFilePath(Path.of("invalid_path.txt"));
+        assertNotEquals(otherConfig, defaultConfig);
     }
 
+    @Test
+    public void hashCode_sameConfig_returnsSameHashCode() {
+        var first = new Config();
```

Comment:
maybe we can change var type to Config? 
```suggestion
        Config first = new Config();
```